docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/java/index.md

```diff
@@ -70,6 +70,8 @@ OTEL_SERVICE_NAME=SERVICE_NAME
 OTEL_RESOURCE_ATTRIBUTES=application=APPLICATION_NAME
 ```
 
+{@import ../../../../../reuse/otel-deployment-environment-tag.md}
```

Comment:
Please put it as a part of point `2` where we do explanation of the configurable things.